### PR TITLE
9.0.3 Release

### PR DESCRIPTION
**Pull request by su-sws-jira**
# DO NOT DELETE

### CODE REVIEW

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

File: composer.json [hunk 1]
```diff
@@ -189,6 +190,7 @@
         "drupal/key": "^1.14",
         "drupal/layout_builder_component_attributes": "^1.1",
         "drupal/layout_builder_modal": "^1.1",
+        "drupal/layout_paragraphs": "^1.0@beta",
```
Comment (sherakama) [inline composer.json:193]: Giving this a go eh? I am interested to hear about how it went at a later date.

Comment (pookmish) [reply composer.json:193]: yeah we're gonna experiment with it some to see how it does :smile:

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr857-xtxzwrld8ehqekpfepokeumd7s19zqrf.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949

Visual Diffs:
- https://dashboard.tugboat.qa/60898ed68e7af630d8caf949/visualdiffs

**Pull request merged**